README.md

```diff
@@ -1,81 +1,41 @@
-# Grants.gov modernization
+# Grants Equity
 
 A modernization effort for Grants.gov.
 
-## Getting Started
+## About the Project
 
-### Agency Mission
+Our vision is for the following to become true:
 
-### Team Mission
+Grants.gov is the simplest, most inclusive, and most gratifying way to find and apply for financial assistance ever built, inside or outside the federal government.
 
-### Project Vision
+Grants.gov helps ensure that no communities are underserved by the federal government.
 
-### Project Information
-<!-- Example Innersource Project Info
- * [Project Website](https://cms.gov/digital-service-cms)
- * [Project Documentation:](https://confluence.cms.gov/)
- * [Project Sprint/Roadmap:](https://jira.cms.gov/)
- * [Project Slack Channel:](https://cmsgov.slack.com/archives/XXXXXXXXXX)
- * [Project Tools/Hosting/Deployment:](https://confluence.cms.gov)
- * Project Keyword(s) for Search: KEYWORD1, KEYWORD2
- * Project Members:
-    * Team Lead, PO, Delivery Lead, Approvers, Trusted Committers etc.
--->
+See [goals.md](./documentation/goals.md) for more information about the vision and goals for the project.
 
-<!-- Example Open Source Info
- * [Project Website](https://cms.gov/digital-service-cms)
- * [Project Documentation:](https://confluence.cms.gov/)
- * Public Contact: opensource@cms.hhs.gov (**NOTE: Do not use individual/personal email addresses**)
- * Follow [@CMSgov](https://twitter.com/cmsgov) on Twitter for updates.
--->
+## Core Team
 
-### Installation
+The core team on the project is a small group of designers, developers, product managers, and more  working for an with the Department of Health and Human Services and other federal agencies.
 
-<!--- Example Install Instructions
+An up-to-date list of core team members can be found in [MAINTAINERS.md](./MAINTAINERS.md). At this time, the project is still building the core team and defining roles and responsibilities. We are eagerly seeking individuals who would like to join the community and help us define and fill these roles. 
```

Comment:
We might need to update ./maintainers.md, do we want to do that in this PR or a separate one?

Reply:
We just updated that file: https://github.com/HHS/grants-equity/blob/main/MAINTAINERS.md#maintainers :)